{"name": "Invalid value for param - coords in the Request Body and getting Incorrect output -- @1.2 ", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [8, 8], 'patches': 0}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "steps": [{"name": "Given I Set POST request api endpoint", "status": "passed", "start": 1628184044750, "stop": 1628184044751}, {"name": "When I Set HEADER param request content type as \"application/json.\"", "status": "passed", "start": 1628184044751, "stop": 1628184044752}, {"name": "When Set invalid request Body { \"roomSize\" : [5,5], \"coords\" : [10, 10], \"patches\" : [[1, 2]], \"instructions\" : \"NEWSNEWS\" }", "status": "passed", "start": 1628184044752, "stop": 1628184044753}, {"name": "And Send a POST HTTP request", "status": "passed", "start": 1628184044753, "stop": 1628184044764}, {"name": "Then I receive error 500, \"Internal Server Error\"", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [8, 8], 'patches': 0}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "start": 1628184044764, "stop": 1628184044765}], "parameters": [{"name": "request_body", "value": "{ \"roomSize\" : [5,5], \"coords\" : [10, 10], \"patches\" : [[1, 2]], \"instructions\" : \"NEWSNEWS\" }"}, {"name": "response_code", "value": "500"}, {"name": "response_error", "value": "Internal Server Error"}], "start": 1628184044749, "stop": 1628184044767, "uuid": "b225771f-1007-47e6-a5f4-1ca58bc56920", "historyId": "fe86b529ec2259e597f3a483085b1e58", "fullName": "Test Imaginary Robot for Valid and Invalid Input values for coords params: Invalid value for param - coords in the Request Body and getting Incorrect output", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Imaginary Robot for Valid and Invalid Input values for coords params"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}